### ISSUE TITLE
LMMS 2.2.1 uses a ton of the cpu on Ubuntu 21.04 on raspberry pi 4

### ISSUE THREAD

**Issue by 2-bithacker** [LMMS/lmms#6171]
Please search the issue tracker for existing bug reports before submitting your own. Delete this line to confirm no similar report has been posted yet.

### Bug Summary
LMMS 2.2.1 uses a ton of the cpu on Ubuntu 21.04 64 bit on raspberry pi 4. I recently installed ubuntu 21.04 desktop on my raspberry pi and LMMS barely works. After configuring the audio server, I tried to play some songs that I had made on an old 32 bit version of lmms-1.1.3 on the same Pi, but running raspberry pi os 32 bit. The songs (being run on lmms 2.2.1) used up a ton of the cpu to where it was super glitchy. It was so bad that my mouse cursor barely worked. It seems like the newer LMMS is a downgrade and has poor function. Does anyone know what could be the problem?

#### Screenshot
x-special/nautilus-clipboard
copy
file:///home/fred/glitch
When I took this, the song was barely playing and cutting out at times; I could barely stop the song and take the screenshot.
#### Affected LMMS versions
2.2.1

**Comment by Veratil** [comment]
I believe you mean version 1.2.2. :)

The CPU usage is known though, there's quite a few other issues opened about it. It's not related just to RPi.

**Comment by 2-bithacker** [comment]
Indeed. 1.2.2. You're right. It seems to use about 25% of the visible cpu on the usage monitor, but it still terribly mispreforms.

**Comment by luzpaz** [comment]
Needs `linux` label
